packages/suite/src/hooks/wallet/useSendForm.ts

```diff
@@ -107,6 +104,17 @@ export const useSendForm = (props: UseSendFormProps): SendContextValues => {
 
     const { control, reset, register, getValues, formState, setValue, trigger } = useFormMethods;
 
+    const values = getValues();
+    const token = values?.outputs?.[0]?.token;
+
+    const fiatRates = props.coins.find(item => {
+        const hasToken = !!token;
+        return (
+            item.symbol === props.selectedAccount.account.symbol &&
+            (!hasToken || item.tokenAddress === token)
+        );
+    });
+
```

Comment:
`useSendForm` is extremely complex, I would avoid adding more lines if not necessary. Could you just create/use a selector which return `CoinFiatRates`?

Then you can remove `coins` from `props`

Reply:
Agree but I don't see anything better than this atm. I will followup on this PR with [this issue](https://github.com/trezor/trezor-suite/issues/10464), where I will refactor how we access rates using `selectFiatRatesByFiatRateKey` so I will make it then easier here too